servers/src/main/java/tachyon/master/file/journal/CompleteFileEntry.java

```diff
@@ -16,39 +16,47 @@
 package tachyon.master.file.journal;
 
 import java.util.List;
-import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
-import com.google.common.collect.Maps;
 
 import tachyon.master.journal.JournalEntry;
 import tachyon.master.journal.JournalEntryType;
 
-public final class CompleteFileEntry implements JournalEntry {
+public final class CompleteFileEntry extends JournalEntry {
   private final List<Long> mBlockIds;
   private final long mId;
   private final long mLength;
   private final long mOpTimeMs;
 
-  public CompleteFileEntry(List<Long> blockIds, long id, long length, long opTimeMs) {
+  @JsonCreator
+  public CompleteFileEntry(@JsonProperty("blockIds") List<Long> blockIds,
+      @JsonProperty("id") long id, @JsonProperty("length") long length,
+      @JsonProperty("operationTimeMs") long opTimeMs) {
     mBlockIds = Preconditions.checkNotNull(blockIds);
     mId = id;
     mLength = length;
     mOpTimeMs = opTimeMs;
   }
 
+  @JsonGetter("blockIds")
```

Comment:
A convention could be to use JsonGetter without an argument.
It would be better for avoiding typos and redundant strings usage.
In this way, some getter method name must be updated in order to keep exactly the same json structure.